examples/text-generation/requirements.txt

```diff
@@ -1,2 +1,5 @@
 datasets
 peft
+transformers_stream_generator
```

Comment:
Not sure if it is right solution to expand this requirement file. As alternative I see adding note in doc, indicating that additional packages required. Please suggest which one is better option